tools/bt2img.py

```diff
@@ -36,12 +36,9 @@
     "Repeat",
     "Subtree",
     "Timeout",
-    "RateController",
-    "DistanceController",
-    "SpeedController",
     "RecoveryNode",
     "PipelineSequence",
-    "RoundRobin"
+    "RoundRobin",
```

Comment:
```suggestion
    "RoundRobin",
    "Control",
```

To keep in line with the other lists.